internal/ls/hover.go

```diff
@@ -407,6 +407,28 @@ func writeComments(b *strings.Builder, comments []*ast.Node) {
 				}
 			}
 			b.WriteString(text)
+		case ast.KindJSDocLinkCode:
```

Comment:
This new code is fine as a temporary placeholder, but leave some `// !!!` comments indicating that this needs to be updated later